### DISCUSSION TITLE
control的dataTamplate

### DISCUSSION THREAD

**Discussion by lejoyke** [BYJRK/DotNet-Discussions#152] [WPF 开发]
![image](https://github.com/BYJRK/DotNet-Discussions/assets/130480804/930d9ec6-b43e-4fac-8ebf-f5481be570ec)

**Comment by lejoyke** [comment]
是不是看不到我写的内容

**Reply by BYJRK** [reply]
我没有看懂你的问题，是不是没写完整？

**Reply by lejoyke** [reply]
我换成了图片了，文字太多它自动会简略，我不太会用

**Comment by BYJRK** [comment]
你可能需要写一个自定义控件，比如一个继承了 `TextBox` 的自定义控件，并为它添加 `Source` 依赖属性；或者你也可以为 `TextBox` 控件书写附加属性；甚至还可以使用现有的一些属性，比如 `Tag`

**Reply by lejoyke** [reply]
好的，谢谢UP
